### PR TITLE
Update packages to resolve many security audit warnings.

### PR DESCRIPTION
**Pull request by rictic**

### CODE REVIEW

File: packages/cli/src/polymer-cli.ts [hunk 1]
```diff
@@ -195,7 +195,9 @@ export class PolymerCli {
         {args: commandArgs});
 
     const commandDefinitions = mergeArguments([command.args, globalArguments]);
-    const commandOptionsRaw = commandLineArgs(commandDefinitions, commandArgs);
+    console.log(`commandLineArgs: ${JSON.stringify(commandLineArgs, null, 2
```
Comment (TimvdLippe) [inline packages/cli/src/polymer-cli.ts]: Leftover debug statement I suppose?

File: packages/polyserve/src/test/cli_test.ts [hunk 2]
```diff
@@ -58,6 +58,8 @@ suite('cli', () => {
       }
     } finally {
       unintercept();
+      console.log('stdout from failed cli invocation:');
+      console.log(stdout);
```
Comment (TimvdLippe) [inline packages/polyserve/src/test/cli_test.ts]: Are these leftover as well?

Comment (rictic) [reply packages/polyserve/src/test/cli_test.ts]: This are intended, the other ones aren't

File: packages/cli/src/polymer-cli.ts (hunk 1 above)
Comment (rictic) [reply packages/cli/src/polymer-cli.ts]: good catch!

**Comment by TimvdLippe** [comment]
We are upgrading to Mocha 5 here, but we are not updating `wct` as well. Reference issue: #227

**Comment by merlinnot** [comment]
So... #377? 😸

**Pull request merged**